.github/workflows/main.yaml

```diff
@@ -73,18 +73,18 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-#      - run: npm publish --access public
-#        env:
-#          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - uses: actions/setup-node@v3
         with:
           registry-url: 'https://npm.pkg.github.com'
           scope: '@rhamzeh'
 
-#      - run: npm publish --access public
-#        env:
-#          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: npm publish --access public
```

Comment:
a bit weird that there are two npm publish...

Reply:
One is for npm, one is for github